packages/compass-connections-navigation/src/navigation-item.tsx

```diff
@@ -184,20 +184,21 @@ export function NavigationItem({
     }
     if (item.type === 'connection') {
       return {
-        'data-is-active': `${isActive}`,
+        'data-is-active': isActive.toString(),
         'data-connection-id': item.connectionInfo.id,
         'data-connection-name': item.name,
+        'data-is-connected': (item.connectionStatus === 'connected').toString(),
```

Comment:
There is unfortunately no easy way for an e2e test to tell if a connection is connected/disconnected. I was using expanded true/false, but it is unreliable because I now also use collapse all to make sure all the connections are rendered to work around the virtual list.

Visually there's a little green dot and the fonts are different and if you hover over it you get different options, but that's tricky to select on in tests.

Reply:
this actually points to an a11y problem, we should probably add an aria-label to the green dot. not sure if that'd be convenient enough for all the e2e tests, but it's for sure not good that we don't have anything beside little visual changes.

Reply:
the green dot is a different element to the connection so would still be a pain because then we'd need separate iterative code to go back up the tree. You can do that in addition to this.

Reply:
that's fair!